packages/core/package.json

```diff
@@ -24,17 +24,16 @@
         "onFileSystem:s3",
         "onFileSystem:s3-readonly"
     ],
-    "main": "./dist/src/extension.js",
+    "main": "./dist/src/extensionNode.js",
     "browser": "./dist/src/extensionWebCore.js",
     "exports": {
         ".": "./dist/src/extension.js",
+        "./node": "./dist/src/extensionNode.js",
         "./web": "./dist/src/extensionWeb.js",
         "./webShared": "./dist/src/extensionWebShared.js",
-        "./extensionCommon": "./dist/src/extensionCommon.js",
-        "./common": "./dist/src/common/index.js",
         "./amazonq": "./dist/src/amazonq/index.js",
         "./codewhisperer": "./dist/src/codewhisperer/index.js",
-        "./codewhispererCommon": "./dist/src/codewhisperer/indexCommon.js",
+        "./codewhisperer/node": "./dist/src/codewhisperer/indexNode.js",
```

Comment:
can we avoid exposing `/node` cases in the library interface (i.e. this "exports" list)? It gives partner teams the wrong idea, and may start to proliferate. 

*Internally*, `node/` and `fooNode.ts` are ways to route to proper implementations. But the interface itself (this "exports" list) shouldn't need to expose that.

Since gumby doesn't support web, that just means gumby won't be called from `extensionWeb.ts`.

Reply:
This separation for node specific was made since indirectly importing gumby code broke web mode. We need to isolate gumby code to `codewhisperer/node` so that we can import `codewhisperer` in to web.

I'll revisit this in the future, openend ticket: IDE-14397